terraform/groups/configurable-api-caller/main.tf

```diff
@@ -140,6 +140,30 @@ resource "aws_lambda_permission" "allow_cloudwatch_dissolutions_rebel1" {
   source_arn    = aws_cloudwatch_event_rule.call_api_caller_lambda_dissolutions_rebel1[0].arn
 }
 
+resource "aws_cloudwatch_event_rule" "call_api_caller_lambda_efs_handle_delayed_submission_sameday" {
+  count               = var.deploy_to == "development" ? 1 : 0
+  name                = "call_api_caller_lambda_efs_handle_delayed_submission_sameday"
+  description         = "Cloudwatch event to call ${aws_lambda_function.configurable_api_lambda.function_name} lambda routinely"
+  schedule_expression = "cron(*/15 7-17 * * MON-FRI *)"
```

Comment:
So quarter past of every hour between 7 and 5 mon-fri. So once an hour, but at quarter past for some reason? Is that the schedule you were going for?

Reply:
I was trying for every 15 minutes.

Reply:
Ah yea! Missed the */ at the start. Cool okay. Looks fine then